specs/interop/predeploys.md

```diff
@@ -150,6 +154,14 @@ function executeMessage(Identifier calldata _id, address _target, bytes calldata
 
 Note that the `executeMessage` function is `payable` to enable relayers to earn in the gas paying asset.
 
+### Deposit Handling
+
+A call to `executeMessage` reverts if the call is made in a [deposit context](./derivation.md#deposit-context).
```

Comment:
We should be particular about the event `ExecutingMessage` rather than a call to the function

Reply:
true! fixed :)